src/components/Display.js

```diff
@@ -15,11 +15,12 @@ export default function Display(props) {
   const charges = calcCharges(props.cart)
   
   const onClick = () => {
-    /*if (props.conditions.apply && config.extra === 'none') return false;*/
-    //if (props.conditions.apply && config.extra === 'static' && config.cart) return client.enhanced_pdp_modal(charges, 'cart');
-    /*if (props.conditions.apply && config.extra === 'static') return client.enhanced_pdp_modal(charges);*/
+    if (props.conditions.vip) window.location.href = process.env.REACT_APP_VIP_UI;
```

Comment:
does the [VIP PR](https://github.com/creditkey/ck-react/pull/63) need to be merged first?